### ISSUE TITLE
Correctly integrate UDTs with stored procedures for DB2

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#192]
This task is a consequence of #162

**Comment by lukaseder** [comment]
https://github.com/jOOQ/jOOQ/issues/164 was closed as won't fix. We're not supporting UDTs for Db2 (at least not until a customer requires it).
